cypress/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "tackle-ui-tests",
```

Assuming it makes sense to add the `cypress` folder to the workspace to share eslint and prettier settings, it'll make sense to use a package name consistent with the other workspaces:


```suggestion
  "name": "@konveyor-ui/ui-tests",
```